src/Symfony/Component/Config/Tests/Resource/ClassExistenceResourceTest.php

```diff
@@ -74,6 +75,22 @@ public function testExistsKo()
         }
     }
 
+    public function testBarParentWithTimestamp()
```

Comment:
Shouldn't `Bar` be `Bad` in regards to the class name itself, `BadParent`?

Same goes for line 88.